dev/com.ibm.ws.anno/test/com/ibm/ws/annocache/test/info/InfoStore_Test.java

```diff
@@ -420,10 +419,6 @@ public void testParameterAndAnnoOrder() {
         Assert.assertEquals(4, parmTypes.size());
         Assert.assertEquals(Arrays.asList("int", String.class.getName(), List.class.getName(), "long"), parmTypes);
 
-        @SuppressWarnings("deprecation")
-        List<List<? extends com.ibm.wsspi.anno.info.AnnotationInfo>> parameterAnnos = mi.getParameterAnnotations();
-        Assert.assertEquals(4, parameterAnnos.size());
-
```

Comment:
Probably going to have to put this back.

Reply:
Changed back.